integrationtest/jms/src/test/java/org/hibernate/search/test/jms/master/JMSMasterTest.java

```diff
@@ -69,6 +73,10 @@
 	 */
 	private static final String CONNECTION_FACTORY_NAME = "java:/ConnectionFactory";
 
+	public static final Poller POLLER = Poller.milliseconds( 10_000, 100 );
+
+	private final QueryParser parser = new QueryParser( "id", TestConstants.stopAnalyzer );
```

Comment:
Right... I'm afraid the problem is widespread though, because I've already seen this pattern (the query parser as a test attribute) multiple times...